airflow-core/newsfragments/59785.significant.rst

```diff
@@ -0,0 +1,6 @@
+``DagBag`` now uses ``Path.relative_to`` for consistent cross-platform behavior
+
+**Backward compatibility:**
+``FileLoadStat`` will no longer produce paths beginning with ``/`` with the meaning of "relative to the dags folder".
```

Comment:
@jedcunningham @ephraimbuddy @Dev-iL -> one more thought here. 

Possibly FileLoadStat here should also contain bundle information here? I think in the current form, it might be ambiguous - if two dags with same path are coming from a different bundle ?

Should we have bundle-id added here/ dag_folder information ?